### PR TITLE
TD-3032 Following right format in naming the export file

### PR DESCRIPTION
**Pull request by sherif-olaboye**
### JIRA link
https://hee-tis.atlassian.net/browse/TD-3032

### Description
I added the name of the delegate from DelegateAccount table and following the standard format in renaming the export file
### Screenshots

image

-----
### Developer checks
(Leave tasks unticked if they haven't been appropriate for your ticket.)

I have:
- [x] Run the formatter and made sure there are no IDE errors (see [info on Text Editor settings](https://hee-tis.atlassian.net/wiki/spaces/TP/pages/3546185813/DLS+Dev+Process) to avoid whitespace changes)
- [ ] Written tests for the changes (accessibility tests, unit tests for controller, data services, services, view models, etc)
- [x] Manually tested my work with and without JavaScript
- [ ] Tested any Views or partials created or changed with [Wave Chrome plugin](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh/related) and addressed any valid accessibility issues
- [ ] Updated/added documentation in [Confluence](https://hee-tis.atlassian.net/wiki/spaces/TP/pages/3546939432/DLS+Code) and/or [GitHub Readme](https://github.com/TechnologyEnhancedLearning/DLSV2/blob/master/README.md). List of documentation links added/changed:
  - [doc_1_here](link_1_here)
- [ ] Updated my Jira ticket with information about other parts of the system that were touched as part of the MR and have to be sanity tested to ensure nothing’s broken
- [x] Scanned over my pull request in GitHub and addressed any warnings from the GitHub Build and Test checks.

### CODE REVIEW

File: DigitalLearningSolutions.Data/DataServices/SelfAssessmentDataService/CandidateAssessmentsDataService.cs [hunk 1]
```diff
@@ -98,7 +98,8 @@ ELSE 0
                             'Supervisor') AS SignOffRoleName,
                         SA.SignOffRequestorStatement,
                         SA.ManageSupervisorsDescription,
-                        CA.NonReportable
+                        CA.NonReportable,
+					 DA.FirstName_deprecated +' '+ DA.LastName_deprecated AS DelegateName
```
Comment (rshrirohit) [inline DigitalLearningSolutions.Data/DataServices/SelfAssessmentDataService/CandidateAssessmentsDataService.cs:102]: Hi Sherif. As the column name suggests, it is deprecated. We now store names in the User table. Please make use of the User table to fetch the names. Also, the join would change accordingly.

**Review by rshrirohit — CHANGES_REQUESTED** [review]
Please use correct table.

File: DigitalLearningSolutions.Data/DataServices/SelfAssessmentDataService/CandidateAssessmentsDataService.cs (hunk 1 above)
Comment (sherif-olaboye) [reply DigitalLearningSolutions.Data/DataServices/SelfAssessmentDataService/CandidateAssessmentsDataService.cs:102]: ok

Comment (sherif-olaboye) [reply DigitalLearningSolutions.Data/DataServices/SelfAssessmentDataService/CandidateAssessmentsDataService.cs]: Done

**Pull request merged**